google-cloud-tools-plugin/src/com/google/gct/idea/CloudToolsPluginInfoService.java

```diff
@@ -21,4 +21,5 @@
  * information service across google-cloud-tools-plugin and google-account-plugin
  */
 public interface CloudToolsPluginInfoService extends PluginInfoService {
+  String getClientVersion();
```

Comment:
This name is going to confuse people who see getPluginVersion().  Since this exists really purely for Debugger invocations I wouldn't mind calling it something like getClientVersionForCloudDebugger() or something like that. (I know that's long..)